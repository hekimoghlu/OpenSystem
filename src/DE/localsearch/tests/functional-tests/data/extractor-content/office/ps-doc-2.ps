%!PS
%%Copyright: copyleft
%%Title: Hello World!
%%Creator: vi
%%CreationDate: 6:00 PM Apr 28, 2024
%%Pages: 1
%%EndComments
/Courier
20 selectfont
72 500 moveto
(Hello world!) show
showpage
