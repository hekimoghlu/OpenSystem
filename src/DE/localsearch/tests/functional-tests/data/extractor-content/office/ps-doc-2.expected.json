{
    "test": {
        "Filename": "ps-doc-2.ps",
        "Comment": "PS document from the office tools"
    },
    "metadata": {
        "@graph": [
	    {
		"nfo:pageCount": 1,
		"nie:copyright": "copyleft",
		"nco:creator": {
		    "nco:fullname": "vi",
		    "@type": "nco:Contact"
		},
		"@type": "nfo:PaginatedTextDocument",
                "nie:contentCreated" : "2024-04-28T18:00:00+0200"
	    }
	]
    }
}
